eng/pipelines/smoke-tests.yml

```diff
@@ -39,7 +41,31 @@ jobs:
           packageType: sdk
           version: "$(DotNetCoreSDKVersion)"
 
-      - script: dotnet run -p ./common/SmokeTests/SmokeTest/SmokeTest.csproj --framework $(TestTargetFramework)
+      - pwsh: Register-PackageSource -Name NightlyFeed -Location $(BlobFeedUrl) -ProviderName Nuget
+        displayName: Register nightly feed package source
+
+      - script: dotnet restore --source '$(BlobFeedUrl)' --source 'https://api.nuget.org/v3/index.json' ./common/SmokeTests/SmokeTest/SmokeTest.csproj
```

Comment:
Why do you need to pass the source explicitly? Shouldn't those get picked up from the nuget.config in the root of the repo?